test/test_simulations.py

```diff
@@ -238,7 +238,7 @@ def test_duration_units_in_parameter_file():
 
 
 def test_distribute_observation(tmp_path):
-    for dtype in (np.float16, np.float32, np.float64, np.float128):
+    for dtype in (np.float16, np.float32, np.float64, np.longdouble):
```

Comment:
I wasn't aware of the existence of `np.longdouble`, which seems an alias for `np.float128`. Since we stopped supporting them because of issue #285 , perhaps it's better to remove them altogether…